appsec/config.go

```diff
@@ -18,22 +18,34 @@ import (
 
 // Configuration environment variables
 const (
-	envAPISecEnabled    = "DD_EXPERIMENTAL_API_SECURITY_ENABLED"
-	envAPISecSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE"
-	envObfuscatorKey    = "DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP"
-	envObfuscatorValue  = "DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP"
-	envWafTimeout       = "DD_APPSEC_WAF_TIMEOUT"
-	envTraceRateLimit   = "DD_APPSEC_TRACE_RATE_LIMIT"
-	envRules            = "DD_APPSEC_RULES"
+	// EnvAPISecEnabled is the env var used to enable API Security
+	EnvAPISecEnabled = "DD_EXPERIMENTAL_API_SECURITY_ENABLED"
+	// EnvAPISecSampleRate is the env var used to set the sampling rate of API Security schema extraction
+	EnvAPISecSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE"
+	// EnvObfuscatorKey is the env var used to provide the WAF key obfuscation regexp
+	EnvObfuscatorKey = "DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP"
+	// EnvObfuscatorValue is the env var used to provide the WAF value obfuscation regexp
+	EnvObfuscatorValue = "DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP"
+	// EnvWAFTimeout is the env var used to specify the timeout value for a WAF run
+	EnvWAFTimeout = "DD_APPSEC_WAF_TIMEOUT"
+	// EnvTraceRateLimit is the env var used to set the ASM trace limiting rate
+	EnvTraceRateLimit = "DD_APPSEC_TRACE_RATE_LIMIT"
+	// EnvRules is the env var used to provide a path to a local security rule file
+	EnvRules = "DD_APPSEC_RULES"
```

Comment:
Okay... call me a pedant if you must, but I feel terribly uncomfortable with the idea of exporting VARIABLES for the entire world to possibly be able to tamper with...

I get these need exposing, but they should be exposed in a way that does not allow overriding values (e.g: functions).